core/domain/fs_domain.py

```diff
@@ -312,7 +446,7 @@ def open(self, filepath):
         self._check_filepath(filepath)
         return self._impl.get(filepath)
 
-    def get(self, filepath):
+    def get(self, filepath: str) -> Sized:
```

Comment:
Why and what is `Sized`?

Reply:
This is because of an error caused by the type annotations of another file 
![image](https://user-images.githubusercontent.com/37413698/158023501-030c2c6e-8125-4d52-861f-c43f2189fa97.png)

Reply:
Just for this reason, you cannot really change it, this should return `FileStream`. Try to resolve the errors in some different way.